basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -2902,6 +2903,117 @@ public static String stripOffQuery(String urlString)
 		return retval;
 	}
 
+	// Copy an LTI content Item from an old site into a new site
```

Comment:
not the proper comment style for methods

Reply:
Agreed